extensions/short-io/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "short-io",
+  "title": "Short.io",
+  "description": "Shorten links using your own domain via Short.io.",
+  "icon": "short-io-icon.png",
```

Comment:
Shouldn't their main icon be this one instead: https://play-lh.googleusercontent.com/BdiGYSn4Dfb7QBhDH0qzmLwMeWUKvIT76RsEh8-B9e30tkh79X82lQxE-X_ExcdldA4t

At least that is the one I see in their website